spec/vmpooler/api/v1_spec.rb

```diff
@@ -417,6 +531,120 @@ def app()
         expect_json(ok = true, http = 200)
       end
 
+      it 'returns multiple VMs even when multiple instances from the same pool are requested' do
+        post "#{prefix}/vm/pool1+pool1+pool2", ''
+
+        expected = {
+          ok: true,
+          pool1: {
+            hostname: [ 'abcdefghijklmnop', 'abcdefghijklmnop' ]
+          },
+          pool2: {
+            hostname: 'qrstuvwxyz012345'
+          }
+        }
+
+        expect(last_response.body).to eq(JSON.pretty_generate(expected))
+
+        expect_json(ok = true, http = 200)
+      end
+
+      it 'returns multiple VMs even when multiple instances from multiple pools are requested' do
+        post "#{prefix}/vm/pool1+pool1+pool2+pool2+pool2", ''
```

Comment:
I could be missing something, but it seems like the only difference between this new block of test and the previous new block of tests is the URL. Does RSpec allow for a test like this to be parameterized so you can tell the test case to run with each valid URL that is expected to produce the same results?